contracts/seed/Seed.sol

```diff
@@ -559,8 +561,8 @@ contract Seed {
                 "Seed: should transfer seed tokens to refund receiver"
             );
         } else {
-            uint256 seedAmountRequired = (hardCap * PRECISION) / price;
-            uint256 feeAmountRequired = (seedAmountRequired * fee) / PRECISION;
+            // uint256 seedAmountRequired = (hardCap * PRECISION) / classes[0].price;
```

Comment:
Why commented, you've already removed them in another PR? Should've just worked on the base of it then

Reply:
removed